### ISSUE TITLE
Cannot resolve field access

### ISSUE THREAD

**Issue by Ivan-Veselov** [javaparser/javasymbolsolver#355]
Task: resolve field access, i.e. constructions like `obj.foo`.
Problem: I'm unable to find a way to do that.

Here is a sample class:
```
public class A {
    public A() {
        B o = new B();
        o.b = 0;
    }
}

class B {
    public int b;
}
```

I expect that from expression `o.b = 0` I can retrieve declaration of field b in class B.

The code I came up with
```
public class Main {
    public static void main(String[] args) throws FileNotFoundException {
        CombinedTypeSolver javaParserTypeSolver = new CombinedTypeSolver();
        javaParserTypeSolver.add(new ReflectionTypeSolver());
        javaParserTypeSolver.add(new JavaParserTypeSolver(new File("src-dir")));

        CompilationUnit unit = JavaParser.parse(new File("src-dir/src.java"));

        unit.accept(
            new VoidVisitorAdapter<Void>() {
                public void visit(FieldAccessExpr n, Void v) {
                    SymbolReference<? extends ResolvedValueDeclaration> ref =
                            JavaParserFacade.get(javaParserTypeSolver).solve(n.getName());

                    if (ref.isSolved()) {
                        System.out.println("Solved");
                    } else {
                        System.out.println("Unsolved");
                    }
                }
            },
        null);
    }
}
```

This code prints "Unsolved" for provided sample.

javaparser-core: 3.5.4
java-symbol-solver-core: 0.6.2

**Comment by Ivan-Veselov** [comment]
According to discussion in #300 it should be a bug. But there is still no issue for it.

**Comment by matozoid** [comment]
Looks like this was fixed in #357
